### PR TITLE
Minor docs fixes

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

File: documentation/pom.xml [hunk 1]
```diff
@@ -16,32 +16,33 @@
     <description>The Infinispan Documentation</description>
 
     <properties>
-        <infinispan.version>8.0</infinispan.version>
-        
+        <infinispan.version>9.0</infinispan.version>
         <maven.deploy.skip>true</maven.deploy.skip>
-		<skipTests>true</skipTests>
+		    <skipTests>true</skipTests>
```
Comment (danberindei) [inline documentation/pom.xml]: Please replace the tabs with spaces.

**Comment by tristantarrant** [comment]
Fixed, @danberindei

File: documentation/pom.xml [hunk 2]
```diff
@@ -16,32 +16,33 @@
...
                 </executions>
                 <configuration>
-                    <toc>true</toc>
+                    <!--backend>docbook</backend-->
                     <backend>html5</backend>
                     <doctype>book</doctype>
-                    <goalPrefix>asciidoctor</goalPrefix>
+                    <preserveDirectories>true</preserveDirectories>
+                    <relativeBaseDir>true</relativeBaseDir>
+                    <sourceDocumentExtensions>adoc</sourceDocumentExtensions>
```
Comment (danberindei) [inline documentation/pom.xml]: I'm getting an error:

    [ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:1.5.3:process-asciidoc (asciidoc) on project infinispan-docs: Unable to parse configuration of mojo org.asciidoctor:asciidoctor-maven-plugin:1.5.3:process-asciidoc for parameter sourceDocumentExtensions: Cannot assign configuration entry 'sourceDocumentExtensions' with value 'adoc' of type java.lang.String to property of type java.util.List -> [Help 1]

And this fixes it:

                    <sourceDocumentExtensions>
                        <sourceDocumentExtension>adoc</sourceDocumentExtension>
                    </sourceDocumentExtensions>

**Comment by tristantarrant** [comment]
Thanks @danberindei , I didn't notice the error scrolling by

**Comment by danberindei** [comment]
Integrated, thanks Tristan!

**Pull request closed without merging**